tests/phpunit/class-wc-stripe-account-test.php

```
@@ -485,8 +485,8 @@ public function test_is_webhook_enabled() {
 	}
 
 	private function clear_webhook_status_cache() {
-		delete_transient( WC_Stripe_Account::TEST_WEBHOOK_STATUS_OPTION );
-		delete_transient( WC_Stripe_Account::LIVE_WEBHOOK_STATUS_OPTION );
+		WC_Stripe_Database_Cache::delete_with_mode( 'webhook_status', 'test' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Database_Cache' in method 'clear_webhook_status_cache'. (undefined)

(StaticAccess)